src/annotator/test/adder-test.js

```diff
@@ -217,6 +219,90 @@ describe('Adder', () => {
     });
   });
 
+  describe('#_findZindex', () => {
```

Comment:
Generally we try to avoid testing code via its private APIs as that can make it more difficult to change the internals of a class.

In this case one way to do that would be to actually check the Z index via the DOM by replacing the `_findZindex` calls with something like:

```js
function getAdderZIndex(left, top) {
  adderCtrl.showAt(left, top);
  return parseInt(adderEl.style.zIndex);
}
```

That would still leave the `_width()` and `_height()` calls but I think that's OK - these would be easy to adapt if the implementation changed.